ansible/scripts/cluster.sh

```diff
@@ -14,6 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-inventory=${INVENTORY:-inventory}
+inventory=${INVENTORY:-../inventory}
 
-ansible-playbook -i ${inventory} cluster.yml $@
+ansible-playbook -i ${inventory} ../playbooks/cluster.yml $@
```

Comment:
As there will be more scripts, I would rather create ``init.sh`` what will set all paths. Something like that:

```shell
$ cat init.sh
#!/bin/bash

INVENTORY_DIR="../inventories"
PLAYBOOKS_DIR="../playbooks"
```

Then:

```shell
$ cat deploy-cluster.sh
#!/bin/bash

. ./init.sh

inventory=${INVENTORY:-${INVENTORY_DIR}/inventory}
ansible-playbook -i ${inventory} ${PLAYBOOK_DIR}/deploy-cluster.yml $@
```